index.bs

```diff
@@ -4902,6 +5009,56 @@ as candidates to be employed in a [=registration ceremony=].
 : Authenticator extension output
 :: None.
 
+
+## Credential Properties Extension (credProps) ## {#sctn-authenticator-credential-properties-extension}
+
+This extension allows [=[WRPS]=] to query additional properties of the [=public key credential source=] created during a 
+[=registration ceremony=] or used during an [=authentication ceremony=].
+
+: Extension identifier
+:: `credProps`
+
+: Operation applicability
+:: [=registration extension|Registration=] and [=authentication extension|Authentication=]
+
+: Client extension input
+:: The Boolean value [TRUE] to indicate that this extension is requested by the [=[RP]=].
+    <xmp class="idl">
+    partial dictionary AuthenticationExtensionsClientInputs {
+        boolean credProps;
+    };
+    </xmp>
+
+: Client extension processing
+:: None, except creating the authenticator extension input from the client extension input.
+
+: Client extension output
+:: Returns the credential properties as discovered by the client, if any, such as when a platform authenticator is used. 
```

Comment:
Did you mean to include a boolean for whether a platform authenticator was used? (&ldquo;such as when a platform authenticator is used&rdquo;)

Reply:
No intention to include a boolean for whether a platform authenticator was used, although this is a perfectly reasonable suggestion and a possible additional property that could be included.  Let me know in next round of review (or via another issue) if you think that should be included as a credential property populated by the client.

The real reason I mentioned a platform authenticator is that generally speaking it's the client that implements or is strongly bound to the platform authenticator code. In that case the client (which is a superset of the authenticator) will have complete knowledge of what it's behaviour is and be the authoritative source for credential properties.

Reply:
I think &ldquo;such as&rdquo; may be imprecise here. What do you think about:

> Returns the credential properties as discovered by the client, if any. For example, a client is especially likely to have knowledge about properties of credentials stored in platform authenticators.

But your call, either way&mdash;I now understand how you intended it.

Reply:
Happy with this re-wording, applying.